src/services/workspaceManager.ts

```diff
@@ -111,6 +112,7 @@ export class WorkspaceFolderContext {
   private _docsLibrary: Thenable<DocsLibrary> | undefined;
   private _ansibleConfig: Thenable<AnsibleConfig> | undefined;
   private _ansibleLint: AnsibleLint | undefined;
+  private _ansibleSyntaxChecker: AnsiblePlaybook | undefined;
```

Comment:
I suppose it's because this is used when the command for checking the syntax is `ansible-playbook --syntax-check`. But I agree that the naming could be better.

Reply:
@tomaciazek, The idea here is, `_ansibleSyntaxChecker` is of type AnsiblePlaybook because it is for `ansible-playbook --syntax-check`. In the near future, for other ansible-playbook related services, extra logic could be added to the AnsiblePlaybook service and those can be used in workpsaceManager with suitable names according to their usage.

Reply:
Either needs to be true:
- `AnsiblePlaybook` will implement those new functionalities, and there is no reason to rename it to something else in the variable
- `AnsiblePlaybook` will not implement those new functionalities, and therefore it should be called differently

Please decide on one only :)

Reply:
Thanks for clarifications. I had something else in my mind. But at this point, having `ansiblePlaybook` as the variable name makes more sense.

Updated!